bigtable/tests/system.py

```diff
@@ -119,14 +119,14 @@ def tearDown(self):
             instance.delete()
 
     def test_list_instances(self):
-        expected = set([instance.name for instance in EXISTING_INSTANCES])
-        expected.add(Config.INSTANCE.name)
-
         instances, failed_locations = Config.CLIENT.list_instances()
-
         self.assertEqual(failed_locations, [])
-        found = set([instance.name for instance in instances])
-        self.assertTrue(expected.issubset(found))
+        # We have added one new instance in `setUpModule`.
+        self.assertEqual(len(instances), len(EXISTING_INSTANCES) + 1)
+        for instance in instances:
+            instance_existence = (instance in EXISTING_INSTANCES or
+                                  instance == Config.INSTANCE)
+            self.assertTrue(instance_existence)
```

Comment:
Please revert: this patch is out-of-sync with `master`.  #5476 hardened this test against simultaneous test runs.